bors.toml

```diff
@@ -0,0 +1,41 @@
+# Bors-ng is a service which provides a merge and review bot for github PRs.
+# When approved for merging (`bors r+`) or test merging (`bors try`), all
+# pending PRs at the time will be merged together in one of two special
+# branches. Either 'staging' or 'trying'.  In the case of `staging` branch,
+# when all status tests pass (see below) the serialized set of merges will become
+# the new destination branch HEAD (i.e. master).  This guarantees there is never
+# any conflicts with PR merge order on the destination branch(es).
+#
+# Note: The branches 'staging.tmp' and 'trying.tmp' must always be ignored
+# by _all_ CI systems.  They are by bors temporarily, and may go away at
+# unpredictable times.
+#
+# Format Ref: https://bors.tech/documentation/#configuration-borstoml
+#
+# status
+# ------------------
+# Selects which tests are required for merging, matching against values
+# from BOTH the older github 'status API' (ref: https://developer.github.com/v3/repos/statuses
+# /#list-statuses-for-a-specific-ref) AND newer 'checks API'. Ref: https://developer.github.com/v3/checks
+# /runs/#list-check-runs-in-a-check-suite both return JSON:
+#
+# Status API:  Matches against '[].context' values
+# Checks API:  Matches against 'check_runs[].name' values
+#
+# Note: The wild-card character '%' is available.
+status = [
+    "cirrus-ci/required/in_podman%",
+    "cirrus-ci/required/testing%",
```

Comment:
This is the line in question.  I was expecting that should match ***ALL*** of the testing tasks, but it only considered the first one.

Reply:
Confirmed. The [docs I found](https://bors.tech/documentation/) explicitly say so:

>Patterns are written using SQL LIKE syntax. For example, this status directive will match **any** CI job that starts with ci/gitlab/:
>
>    status = [ 'ci/gitlab/%' ]

(emphasis mine)

Reply:
so def. a bug of some sort then.